src/export/digitalPlanning/model.ts

```diff
@@ -482,11 +484,12 @@ export class DigitalPlanning {
         flow: this.flow as LooseFlowGraph,
       });
       const { flag } = Object.values(result)[0];
+      const title = [flag.category, flag.text].join(" / ");
 
       return [
         {
-          value: [flag.category, flag.text].join(" / "),
-          description: flag.description,
+          value: title,
+          description: this.findDescriptionFromValue("ResultFlag", title), // flag.description may be custom text
```

Comment:
This still won't validate successfully if an application is also using custom flag _titles_ - but it should catch more cases and I'll keep adjusting as we work on flags logic & multiple results overall. I'm hesitant to remove the enum type for "ResultFlag" entirely yet.